release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -612,37 +612,50 @@ With this release, the Performance Addon Operator (PAO) must-gather image is no
 For further information about gathering debugging information relating to low-latency tuning, see xref:../scalability_and_performance/cnf-low-latency-tuning.adoc#cnf-collecting-low-latency-tuning-debugging-data-for-red-hat-support_cnf-master[Collecting low latency tuning debugging data for Red Hat Support].
 
 [id="ocp-4-14-NRO-image-must-gather"]
-==== Collecting data for the NUMA Resources Operator with the must-gather image of the operator
+==== Collecting data for the NUMA Resources Operator with the must-gather image of the Operator
 
-In this release, the `must-gather` tool is updated to collect the data of the NUMA Resources Operator with the `must-gather` image of the operator.
+In this release, the `must-gather` tool is updated to collect the data of the NUMA Resources Operator with the `must-gather` image of the Operator.
 
 For further information about gathering debugging information for the NUMA Resources Operator, see xref:../scalability_and_performance/cnf-numa-aware-scheduling.html#cnf-about-collecting-nro-data_numa-aware[Collecting NUMA Resources Operator data].
 
 [id="ocp-4-14-additional-power-savings-control"]
 ==== Enabling more control over the C-states for each pod
 
-With this release, you have more control over the C-states for your pods. Now, instead of disabling C-states completely, you can specify a maximum latency in microseconds for C-states. You can configure this option in the `cpu-c-states.crio.io` annotation, which helps to optimize power savings in high-priority applications by enabling some of the shallower C-states instead of disabling them completely.
+With this release, you have more control over the C-states for your pods. Now, instead of disabling C-states completely, you can specify a maximum latency in microseconds for C-states. You can configure this option in the `cpu-c-states.crio.io` annotation. This helps to optimize power savings in high-priority applications by enabling some of the shallower C-states instead of disabling them completely.
```

Comment:
I found both `C-states` and C-states in the repo so I'm not sure if back ticks are necessary or not. I wanted to bring it to your attention though.

Reply:
`C-states` is incorrect in this instance.